LICENSE

```diff
@@ -1,7 +1,7 @@
 Node.js is licensed for use as follows:
 
 """
-Copyright Node.js contributors. All rights reserved.
+Copyright Joyent, Inc. and other Node.js contributors. All rights reserved.
```

Comment:
Does this block need to be updated? Immediately following it is 
```
This license applies to parts of Node.js originating from the https://github.com/joyent/node repository:
```
which is almost identical and already includes the Joyent attribution.